engine/org.eclipse.birt.report.engine.emitter.html/src/org/eclipse/birt/report/engine/emitter/html/HTMLReportEmitter.java

```diff
@@ -2661,9 +2661,13 @@ public void startNode(Node node, HashMap<?, ?> cssStyles) {
 
 				if (attrValue != null) {
 					if ("img".equalsIgnoreCase(nodeName) && "src".equalsIgnoreCase(attrName)) {
-						String attrValueTrue = handleStyleImage(attrValue).getUri();
-						if (attrValueTrue != null) {
-							attrValue = attrValueTrue;
+						BackgroundImageInfo img = handleStyleImage(attrValue);
+						String attrValueTrue = null;
+						if (img != null) {
+							attrValueTrue = handleStyleImage(attrValue).getUri();
```

Comment:
I think I'd prefer reusing `img` here instead of calling `handleeStyleImage` again.

Reply:
Confirmed, I will change it.